go/libkb/pvl_helpers.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package libkb
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/PuerkitoBio/goquery"
+	keybase1 "github.com/keybase/client/go/protocol"
+	jsonw "github.com/keybase/go-jsonw"
+)
+
+func pvlStringToService(service string) (keybase1.ProofType, ProofError) {
+	switch service {
+	case "twitter":
```

Comment:
in the past, I went pretty far out of the way to put everything about the Foo services into `proof_support_foo`.  Can we still achieve that?